python_modules/airline-demo/airline_demo_tests/test_types.py

```diff
@@ -0,0 +1,44 @@
+import sys
+import tempfile
+
+from airline_demo.resources import spark_session_local, tempfile_resource
+from airline_demo.types import SparkDataFrameSerializationStrategy
+
+
+class MockResources(object):
+    def __init__(self, spark, tempfile_):
+        self.spark = spark
+        self.tempfile = tempfile_
+
+
+class MockPipelineContext(object):
```

Comment:
a bit concerned that we are relying on there being no type checks in order for this to work